### DISCUSSION TITLE
Assistants feature not working

### DISCUSSION THREAD

**Discussion by kneelesh48** [danny-avila/LibreChat#2569] [Troubleshooting]
Assistants feature is not working in the latest version.
It not able to load open ai assistants that already exist and creating new assistants gives 500 internal server error

**Comment by danny-avila** [comment]
No logs, no context, there is not much I can do to help you. It's working on my end:

![image](https://github.com/danny-avila/LibreChat/assets/110412045/edc07258-9187-49f8-aba7-09857d683cb2)
![image](https://github.com/danny-avila/LibreChat/assets/110412045/faa5e23f-8fea-4bb0-b6f3-0ae0ea80b3d2)

There are some issues on OpenAI's side that is on their end that was reported: https://community.openai.com/t/invalid-model-gpt-3-5-does-not-support-image-message-content-types/728895

But other than this, everything working as expected

**Comment by kneelesh48** [comment]
I changed the openai api key but forgot to change the assistants api key
